glib/src/subclass/object.rs

```diff
@@ -36,7 +36,7 @@ pub trait ObjectImpl: ObjectSubclass + ObjectImplExt {
     ///
     /// This is called whenever the property value of the specific subclass with the
     /// given index should be returned.
-    fn get_property(&self, _obj: &Self::Type, _id: usize, _pspec: &ParamSpec) -> Value {
+    fn property(&self, _obj: &Self::Type, _id: usize, _pspec: &ParamSpec) -> Value {
```

Comment:
This would be another such case. `fn property()` looks a bit strange to me at least, but `fn get_property()` seems to be nicer.

Not sure what the pattern here would be... `get_` prefix included if more than the `&self` parameter, and for subclass code more than the `&self, &Self::Type` parameters?

Reply:
Well, there are many cases for which I think it's still preferable to elide `get`, like for the `ActionGroup::action_state` example above, or the many examples of `child_by_name`, `child_by_index`, `preferred_width`, etc... Like I mentioned in the `gir` PR, I believe now that *not* removing `get` is more the exception than the rule. I came to that conclusion because starting from round 1, I proceeded to remove the remaining `get` on a case by case basis and I found myself removing most of them. I browsed the result of the `get` removal from this round several time (and lost monthes of sight in the process :)). Now, I think deciding for the few for which `get` should be restored will be very subjective.

So what we need to choose, and by *we*, I mean you three :) since I already made my choice in these PR, is:

* Do we revert to round 1 and only apply subclass changes.
* Someone also browses through the changes in function definitions (the regen and fix-getters-def commits) and picks the elisions that should be reverted.
* We revert `get_object` & `get_property` and keep it like this until something else shows up.

Reply:
Forgot to mention that I'd preferred the 2d option :)

Reply:
I'd like to move forward with this. Since yesterday, I proceeded to check some crates for usage (see the end of [this discussion](https://github.com/gtk-rs/gtk-rs/pull/455#pullrequestreview-638212421)), but there doesn't seem to be a defined convention. If someone find courage to browse at least some of the changes in the regen and fix-getters-def commits, that would be helpful and we could discuss, otherwise I would adopt option 1.

Reply:
The cautious change would be 1), but IMHO this can also land as is. The difference in the remaining functions that are renamed here (outside `src/subclass`) is really small and I'd probably tell you a different opinion if you ask me on two different days.

We can do either depending on what others think. In the worst case we revert some functions again later, but IMHO in sum this is a positive change.

@bilelmoussaoui @GuillaumeGomez ?

Reply:
> The difference in the remaining functions that are renamed here (outside src/subclass) is really small

I'll update the commit stack later today so that subclass related changes are separated from the others, that would ease reviewing and identification of potential candidates for `get` restore.

Reply:
I think it's fine to rename the get_property to just property, it's a bit confusing at first but I think that comes from the habit of using `get_property` here.

Reply:
Ok, so it seems everybody agrees.

Reply:
@bilelmoussaoui do you still want to keep `get` for `get_object` though?

Reply:
@fengalin nop, let's remove it :)